composer.json

```diff
@@ -11,7 +11,8 @@
         "php-http/guzzle7-adapter": "^0.1.1",
```

Comment:
The Guzzle, PSR dependencies here can be removed and replaced with [`psr/http-client-implementation`](https://packagist.org/providers/psr/http-client-implementation).

Can add Guzzle into `require-dev` for the tests.